From d11f3f72feff415a679f8fa0a79c89bdbc00a9a6 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Sat, 17 Dec 2016 11:25:24 -0500
Subject: [PATCH] CMake: Do not install documentation

As RPM has its own mechanism for installing documentation,
we don't need CMake to do it for us.
---
 CMakeLists.txt | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1c55e57..8e54834 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -291,16 +291,3 @@ if(APPLE)
     install(FILES ${CMAKE_SOURCE_DIR}/Dist/OSX/Lugaru.icns DESTINATION ${LUGARU_RESDIR})
     install(FILES ${CMAKE_SOURCE_DIR}/Dist/OSX/Info.plist DESTINATION ${LUGARU_APP_ROOT}/Contents)
 endif(APPLE)
-
-# Documentation
-
-install(FILES ${CMAKE_SOURCE_DIR}/AUTHORS
-              ${CMAKE_SOURCE_DIR}/CONTENT-LICENSE.txt
-              ${CMAKE_SOURCE_DIR}/COPYING.txt
-              ${CMAKE_SOURCE_DIR}/README.md
-              ${CMAKE_SOURCE_DIR}/RELEASE-NOTES.md
-              ${CMAKE_SOURCE_DIR}/Docs/DEVTOOLS.txt
-              ${CMAKE_SOURCE_DIR}/Docs/README.Empire.txt
-              ${CMAKE_SOURCE_DIR}/Docs/README.SevenTasks.txt
-              ${CMAKE_SOURCE_DIR}/Docs/README.Temple.txt
-        DESTINATION ${LUGARU_DOCDIR})
-- 
2.9.3

